packages/block-editor/src/store/private-actions.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * Returns an action object used in signalling that the block interface, eg. toolbar, outline, etc. should be hidden.
+ *
+ * @return {Object} Action object.
+ */
+export function __experimentalHideBlockInterface() {
```

Comment:
I think we settled that we don't need to have the `__experimental` prefix anymore, since we lock them. That goes to the other places as well, like selectors, etc..

Reply:
How does that affect things like the documentation generation, which is currently configured to skip functions with the __experimental prefix?

Reply:
I removed the prefix in [f8e6574](https://github.com/WordPress/gutenberg/pull/47375/commits/f8e65749d3ef4ad8f70ee6553eca043894b1752b), but if we decide we want to change it back that commit can be reverted 👍